Modules/_sqlite/module.c

```diff
@@ -142,29 +142,29 @@ pysqlite_register_adapter_impl(PyObject *module, PyTypeObject *type,
 /*[clinic input]
 _sqlite3.register_converter as pysqlite_register_converter
 
-    name as orig_name: unicode
-    converter as callable: object
+    type as tp: unicode
+    callable as converter: object
     /
 
-Registers a converter with sqlite3.
+Register a function to convert SQLite types to Python types.
```

Comment:
Maybe this is better:

```suggestion
Register a function to convert SQLite values to Python objects.
```

Reply:
Agree, I think it's better with this change.